compiler/rustc_mir_transform/src/pass_manager.rs

```diff
@@ -173,14 +171,5 @@ pub fn dump_mir_for_pass<'tcx>(
 }
 
 pub fn dump_mir_for_phase_change<'tcx>(tcx: TyCtxt<'tcx>, body: &Body<'tcx>) {
-    let phase_index = body.phase.phase_index();
-
-    mir::dump_mir(
-        tcx,
-        Some(&format_args!("{:03}-000", phase_index)),
```

Comment:
... but this one has changed behaviour, I think? `dump_file_basename` does this:
```rust
format!(".{:03}-{:03}", body.phase.phase_index(), body.pass_count)
```
so the digits after the `-` won't be `000`? Unless `phase_index` is somehow always 0 here?